pypeclub/pype
# Task to version use settings

## Changes
- Task to Version status event handler use project settings
- it is possible to set limitations of asset types by short names in list !this works the opposite way as in `VersionToTaskStatus`!
    - in `VersionToTaskStatus` entered asset types are skipped here are skipped asset types not entered to list
    - if list is empty limitation is turned off
    - `IDEA`: Radio button in settings to Exclude/Include entered asset types?

- `join_query_keys` removed from event handlers as method is implemented in `BaseHandler`

## How to test
- go to file `~/pype/modules/ftrack/events/event_task_to_version_status.py` on line 20 (in method `is_event_invalid`) and skip the method with `return`
    - the method check if status change was done with user that is running the ftrack server
- disable Version to Task status event handler (otherwise will trigger change back to task -> infinite loop of status changes)
